tests/namespace/test_sensor.py

```diff
@@ -231,5 +231,5 @@ def test_getpdu_xcvr_tx_bias_sensor(self):
 
         # test for each channel
         for channel in (2, 4):
-            self._test_getpdu_xcvr_sensor(get_transceiver_sensor_sub_id(self.IFINDEX, SENSOR_TYPE_PORT_TX_BIAS + channel)[0], expected_values)
+            self._test_getpdu_sensor(get_transceiver_sensor_sub_id(self.IFINDEX, SENSOR_TYPE_PORT_TX_BIAS + channel)[0], expected_values)
```

Comment:
Please add the new test cases in namespace/test_sensor as well.

Reply:
added.